src/roles/vsd-decouple/tasks/main.yml

```diff
@@ -63,9 +63,12 @@
     pause:
       seconds: 10
 
-  - name: Execute list_p1db command on node3
-    shell: "{{ p1db_cmd }}"  # noqa 305
+  - name: Wait until VSD is decoupled
+    shell: "{{ p1db_cmd }}"
```

Comment:
can you add back the ansible lint exception

Reply:
Done